app.py

```diff
@@ -52,10 +52,9 @@ def api_post():
     data = flask.request.json
     print(data)
     username = _validate_guest(data['userKey'])
-    if username == data['userName']:
-        return get_method(data['method'])(data)
-    else:
+    if username is None or username != data['userName]
```

Comment:
hang on, I think this is a syntax error :)

Reply:
Agh! That's what I get for using the web-editor